packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -1749,17 +1749,25 @@ def get(self, version):
         return success_response(premium_tracks)
 
 
+full_nft_gated_track_signatures_response = make_full_response(
+    "nft_gated_track_signatures_response",
+    full_ns,
+    fields.Nested(nft_gated_track_signature_mapping),
+)
+
+
 @full_ns.route("/<string:user_id>/nft-gated-signatures")
 class NFTGatedTrackSignatures(Resource):
     @record_metrics
     @full_ns.doc(
-        id="""Get Gated Track Signatures""",
+        id="""Get NFT Gated Track Signatures""",
```

Comment:
Technically, this changes the name of the SDK method. But since it was previously broken (return type of void), I don't think that's a big deal.

Reply:
Yup, I highly doubt anyone was using this